hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/RequestHandler.java

```diff
@@ -78,6 +79,10 @@ public class RequestHandler {
   private final BaseFileHandler dataFileHandler;
   private final MarkerHandler markerHandler;
   private final Registry metricsRegistry = Registry.getRegistry("TimelineService");
+  // This read-write lock is used for syncing the file system view if it is behind client's view
```

Comment:
We have locking at two levels now. Here. and then inside AbstractTableFileSystemView.  Could we simplify this.

Reply:
This read-write lock is removed now.